src/components/map/map-control.js

```diff
@@ -305,18 +319,21 @@ const MapControlFactory = () => {
       onToggleSplitMap: PropTypes.func.isRequired,
       onToggleMapControl: PropTypes.func.isRequired,
       onSetEditorMode: PropTypes.func.isRequired,
+      onToggleEditorVisibility: PropTypes.func.isRequired,
       top: PropTypes.number.isRequired,
 
       // optional
       readOnly: PropTypes.bool,
       scale: PropTypes.number,
       mapLayers: PropTypes.object,
-      editor: PropTypes.object
+      editor: PropTypes.object,
+      indexMap: PropTypes.number
     };
 
     static defaultProps = {
       isSplit: false,
-      top: 0
+      top: 0,
+      indexMap: 0
```

Comment:
`index` instead of `indexMap`

Reply:
replace into mapIndex=0

Reply:
did you push? it'still showing `indexMap: 0`